Modules/Loadable/Markups/MRML/vtkMRMLMarkupsDisplayNode.h

```diff
@@ -351,6 +351,14 @@ class  VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkMRMLMarkupsDisplayNode : public
   static const char* GetCurveLineSizeModeAsString(int mode);
   static int GetCurveLineSizeModeFromString(const char*);
 
+  /// Set variant radius for tube filter.
+  /// Values are given from 'VTK_VARY_RADIUS_' macro
+  vtkSetMacro(VaryRadius, int);
```

Comment:
It should be something like `LineDiameterMode` (as we already have a `LineDiameter` member that controls the size).

Reply:
Ok, I think I can introduce `enum LineDiameterMode{ Constant = VTK_VARY_RADIUS_OFF(0), VaryByScalar = VTK_VARY_RADIUS_BY_SCALAR(1)  }`. There is also `VTK_VARY_RADIUS_BY_VECTOR(2), VTK_VARY_RADIUS_BY_ABSOLUTE_SCALAR(3), VTK_VARY_RADIUS_BY_VECTOR_NORM(4)` but I think they are unnecessary for now, maybe for the future